### PR TITLE
[KAN-9]:Added Routing in the application and created placeholder components

### PR DESCRIPTION
**Pull request by poulami-saha**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,71 +1,27 @@
-import logo from './logo.svg';
-import './App.css';
```
Comment (joanwolk) [inline src/App.js:2]: please add back this CSS file so the app imports and uses the Livvic font by default

File: src/components/Layout/Layout.module.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+  }
+  
+  .content {
+    flex: 1;
+    padding: 20px;
+    margin-block: 40px;
+  }
```
Comment (joanwolk) [inline src/components/Layout/Layout.module.css:10]: To have the content of the page fit neatly (no whitespace, all text fully visible), you should remove the padding and margin-block from `.content` and add the following rule:

```
.content > :first-child {
  padding-bottom: 20px;
  margin-block-start: 75px;
}
```

File: yarn.lock
Comment (joanwolk) [inline yarn.lock:1]: Are we using `npm` or `yarn` for this project? So far, we've been using npm. If that's what we plan to continue, please remove this file and run `npm install` to update the existing `package-lock.json` instead.

**Review by joanwolk — CHANGES_REQUESTED** [review]
Having both `package-lock.json` and `yarn.lock` is confusing and I think we should be clear about which one we're using—which is to say, are we using npm or yarn?

File: yarn.lock
Comment (poulami-saha) [reply yarn.lock:1]: lets continue with npm, its my practice of using yarn

**Review by joanwolk — APPROVED** [review]

**Pull request merged**